src/Infrastructure/TemplateBasedMailer.php

```diff
@@ -36,4 +36,17 @@ public function sendMail( EmailAddress $recipient, array $templateArguments = []
 		);
 	}
 
+	/**
+	 * @throws \RuntimeException
+	 */
+	public function sendMailToOperator( EmailAddress $replyToAddress, array $templateArguments = [] ) {
```

Comment:
I had a proper look at this in my IDE now.

The instance on which `sendMailToOperator` is being called only has that method being called upon it. And all other instances of `TemplateBasedMailer` do not have `sendMailToOperator` called on it. That means there is no reason to put them in the same interface. (See also: https://en.wikipedia.org/wiki/Interface_segregation_principle)

How about creating an `OperatorMailer` or `TemplateBasedOperatorMailer` that is like the existing `TemplateBasedMailer` but only has the operator method? An alternative approach is to keep them in the same class but have two interfaces, though this is way more disruptive, as you'd need to update existing type hints to one of the interfaces.

Btw, I'm not so sure about the `TemplateBased` part in the name. The implementation sure depends on a twig template, but from the user perspective, the interface just takes a bunch of arguments. I'd be nicer for the name to reflect the second part. `ArgumentTakingMailer` is somehow weird tho.